doc/source/_templates/indexsidebar.html

```diff
@@ -1,4 +1,5 @@
             <h3>Resources</h3>
             <ul>
+              <li><a href="https://numpy.org/">NumPy.org website</a></li>
               <li><a href="https://scipy.org/">Scipy.org website</a></li>
             </ul>
```

Comment:
This only shows up on the index.html page

Reply:
just checking, it shows up at `numpy.org/doc` index.html as well? otherwise it's just a link to self

Reply:
The index.html for each version's documentation, so
- https://numpy.org/devdocs/index.html
- https://www.numpy.org/doc/1.17/index.html
- https://www.numpy.org/doc/1.16/index.html

etc. The non-devdoc ones will need to be rebuilt to get this change.